pkg/security/secl/model/process_cache_entry.go

```diff
@@ -59,6 +59,16 @@ func (pc *ProcessCacheEntry) GetNextAncestorNoFork() *ProcessCacheEntry {
 	return ancestor
 }
 
+// HasCompleteLineage returns false if, from the entry, we cannot ascend the ancestors list to PID 1
+func (pc *ProcessCacheEntry) HasCompleteLineage() bool {
+	if pc.Pid == 1 { // we reach pid 1, the lineage is complete
```

Comment:
I think we could convert it to a non-recursive version ?

Reply:
Updated to non-recursive on last commit